services/brig/src/Brig/User/Search/Index.hs

```diff
@@ -50,7 +50,7 @@ module Brig.User.Search.Index
 where
 
 import Brig.Data.Instances ()
-import Brig.Index.Types (CreateIndexSettings (..))
+import Brig.Index.Types (CreateIndexSettings (..), UncheckedLegacyAuthId (..))
```

Comment:
I yhink can't do proper parse error handling, because the parsing happens in a `Cql` instance in DB lookup. This happens in a batch operation, so I'm not sure if it's possible to skip rows here.